### PR TITLE
MDS-2140 GenTax Export + Update NoW is_historical logic

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: services/core-api/app/api/exports/mines/resources/mine_summary_resource.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from app.api.utils.access_decorators import requires_role_view_all
+from app.api.constants import MINE_DETAILS_JSON, TIMEOUT_60_MINUTES
+
+
+class MineSummaryResource(Resource):
+    @api.doc(
+        description=
+        'Returns a subset of mine data. Column headers: mine_guid, mine_name, mine_no, mine_region, major_mine_ind, operating_status, operating_status_code, effective_date, tenure, tenure_code, commodity, commodity_code, disturbance, disturbance_code, permit_no, permittee_party_name'
+    )
+    #@requires_role_view_all
```
Comment (exzizt) [inline services/core-api/app/api/exports/mines/resources/mine_summary_resource.py:19]: Needed?

File: services/core-api/app/api/exports/mines/resources/mine_summary_resource.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    def get(self):
+
+        json_string = None   #cache.get(MINE_DETAILS_JSON)
```
Comment (exzizt) [inline services/core-api/app/api/exports/mines/resources/mine_summary_resource.py:22]: Re-enable?

**Review by exzizt — CHANGES_REQUESTED** [review]
What the heck **_IS THIS_**?!

File: services/core-api/app/api/exports/mines/resources/mine_summary_resource.py (hunk 2 above)
Comment (nayr974) [reply services/core-api/app/api/exports/mines/resources/mine_summary_resource.py]: Re-enabled :)

File: services/core-api/app/api/exports/mines/resources/mine_summary_resource.py (hunk 1 above)
Comment (nayr974) [reply services/core-api/app/api/exports/mines/resources/mine_summary_resource.py]: Yes, added :)

**Review by exzizt — DISMISSED** [review]

**Review by mhuseinov — APPROVED** [review]

File: migrations/sql/R__view_mine_summary.sql [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+CREATE OR REPLACE VIEW public.mine_summary_view
+AS SELECT m.mine_guid::character varying AS mine_guid,
+    p.permit_guid::character varying AS permit_guid,
+    m.mine_name,
+    m.mine_no AS mine_number,
+    m.latitude::character varying AS mine_latitude,
+    m.longitude::character varying AS mine_longitude,
+    efs.exemption_fee_status_code AS mine_exemption_fee_status_code,
+    efs.description AS mine_exemption_fee_status_d,
```
Comment (exzizt) [inline migrations/sql/R__view_mine_summary.sql:11]: "mine_exemption_fee_status_d"? Why not "mine_exemption_fee_status_description"?

**Review by exzizt — APPROVED** [review]
✔️

**Pull request merged**